string name
geometry_msgs/Point[] features
float32 detectionTolerance
float32 placementUncertainty
